frontend/src/pages/Home.jsx

```
@@ -171,11 +195,28 @@ const Home = ({ user, onLogout, darkMode, toggleDarkMode }) => {
         break;
+      case "friends":
+        navigate("/friends");
+        break;
       default:
         navigate("/");
     }
   };
 
+  if (loadingChats) return (
+    <div className="flex justify-center items-center h-screen bg-gray-50 dark:bg-slate-900">
+      <div className="animate-spin h-12 w-12 border-4 border-indigo-600 border-t-transparent rounded-full"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)